io/EptReader.cpp

```diff
@@ -215,12 +216,19 @@ void EptReader::initialize()
     // Set geometry from polygons.
     if (m_args->m_polys.size())
     {
-        m_polys.clear();
+        m_queryPolys.clear();
         for (Polygon& poly : m_args->m_polys)
         {
             // Throws if invalid.
             poly.valid();
```

Comment:
valid just returns a bool.  It doesn't throw.

https://github.com/PDAL/PDAL/blob/master/pdal/Geometry.cpp#L176